cmake/OpenEXRSetup.cmake

```diff
@@ -40,7 +40,8 @@ set(IEX_NAMESPACE "Iex" CACHE STRING "Public namespace alias for Iex")
 option(OPENEXR_INSTALL_PKG_CONFIG "Install OpenEXR.pc file" ON)
 
 # Whether to enable threading. This can be disabled, although thread pool and tasks
-# are still used, just processed immediately
```

Comment:
@kdt3rd, I think the existing comment `although thread pool and tasks are still used` is no longer true, right? Even if ENABLE_THREADING is ON, then no threadpool is created unless setGlobalThreadCount(n) is called with nonzero n, but OpenEXR will still be thread-safe.  I think this what was originally wanted in #1902. ENABLE_THREADING should only be set to OFF on systems which do not support threading

Reply:
oh, it is still true technically, in that there is a "default" thread pool that has no threads running. And the writing classes will still add tasks to that threadpool until the next significant release even when threads == 0 (where when threads == 0, the reading classes don't incur that overhead. However yes, ENABLE_THREADING should be on for 99% of use cases, especially now that pthread and such are just part of libc on most systems, and only reserved for use on systems where no threading primitives are available, or threads are guaranteed to not be employed as it makes the library non-reentrant and non-threadsafe